engine-standalone-storage/src/relayer_db/mod.rs

```diff
@@ -0,0 +1,258 @@
+use aurora_engine::engine;
+use aurora_engine::transaction::EthTransactionKind;
+use aurora_engine_sdk::env::{self, Env};
+use aurora_engine_sdk::promise;
+use aurora_engine_types::account_id::AccountId;
+use aurora_engine_types::H256;
+use postgres::fallible_iterator::FallibleIterator;
+
+use crate::{BlockMetadata, Storage};
+
+pub mod types;
+
+const TRANSACTION_QUERY: &str = "
+SELECT
+  transaction.block, transaction.index, transaction.id,
+  transaction.hash, transaction.near_hash, transaction.near_receipt_hash,
+  transaction.from, transaction.to, transaction.nonce, transaction.gas_price,
+  transaction.gas_limit, transaction.gas_used, transaction.value, transaction.input,
+  transaction.v, transaction.r, transaction.s, transaction.status, transaction.output,
+  block.hash as block_hash
+FROM transaction INNER JOIN block
+ON transaction.block = block.id
+ORDER BY transaction.block, transaction.index
```

Comment:
The reason I can't do `SELECT *` here to simply get all the columns is because of the name collision on "hash" (both tables have a column with this name). So I need to explicitly write what columns I want and rename the hash column coming from the block table.

Reply:
Ahh makes sense.